src/Categories/Object/Biproduct.agda

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+
+-- Biproducts, a-la Karvonen.
+--
+-- This definition has advantages over more traditional ones,
+-- namely that that we don't require either enrichment in CMon/Ab, or Zero Objects.
+--
+-- See https://arxiv.org/abs/1801.06488
+module Categories.Object.Biproduct {o ℓ e} (𝒞 : Category o ℓ e) where
```

Comment:
can we define it using `IsProduct` and `IsCoproduct`? in that way, we then obtain both predicate and structure style and you get to reuse properties of products and coproducts.

Reply:
Great call! That would make things a lot simpler.

Reply:
In a similar vein, `IsBiproduct` would probably be handy.

Reply:
I just realized that there isn't `IsCoproduct` but it's not something difficult to define.

Reply:
Spun out to #250, with comments addressed.